contracts/contracts/solidity/interfaces/IStarknetCore.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Let's replace this interface with the one from the source: https://github.com/starkware-libs/starkgate-contracts/blob/main/src/starkware/starknet/solidity/IStarknetMessaging.sol